real_preprocessing/README.md

```diff
@@ -0,0 +1,24 @@
+# real\_preprocessing
+
+### Additional packages  
+
+* [apriltag\_ros](http://wiki.ros.org/apriltag_ros)
+* [cv\_bridge](http://wiki.ros.org/cv_bridge)
+* [tf\_conversions](http://wiki.ros.org/tf_conversions)
+
+[OpenCV](https://www.learnopencv.com/install-opencv-3-4-4-on-ubuntu-16-04/) was installed on Ubuntu 16.04 and included in CMakeLists.txt as follows
+
+	find_package(find_package(OpenCV REQUIRED PATHS "YOUR-FOLDER-PATH")
+
+### Image publishing 
+**corner\_detection.cpp** simulates image capture with user input. By default, `""` /`ENTER` is the assigned trigger for image processing and YAML dumping. The YAML file for each frame detection is dumped at the working directory. To view the video stream detection, [image\_view](http://wiki.ros.org/image_view) was the library included. Rviz can be used alternatively with some configuration.
+
+### Camera calibration
+The package requires calibration of the camera to be used. This was done using [camera\_calibration](http://wiki.ros.org/camera_calibration).  The intrinsics obtained should be recorded as it is required for detection accuracy. Your intrinsic matrix, K, must be specified in **corner\_detection.cpp**, specifically at line 18,  `intrinsic << #, #, #, #, #, #, #, #, #;`.
+
+### Steps: 
```

Comment:
Lets try looking into implementing this in a roslaunch file. (This is lower priority)